JsonSchema.DataGeneration/JsonSchemaExtensions.cs

```diff
@@ -79,8 +80,23 @@ internal static GenerationResult GenerateData(this RequirementsContext requireme
 	}
 
 	private static readonly IEnumerable<IRequirementsGatherer> _requirementsGatherers =
-		typeof(IRequirementsGatherer).Assembly
-			.DefinedTypes
+			new Type[]
+			{
+				typeof(AllOfRequirementsGatherer),
+				typeof(AnyOfRequirementsGatherer),
+				typeof(ConditionalRequirementsGatherer),
+				typeof(ConstRequirementsGatherer),
+				typeof(ContainsRequirementsGatherer),
+				typeof(EnumRequirementsGatherer),
+				typeof(FalseRequirementsGatherer),
+				typeof(ItemsRequirementsGatherer),
+				typeof(NotRequirementsGatherer),
+				typeof(NumberRequirementsGatherer),
+				typeof(OneOfRequirementsGatherer),
+				typeof(PropertiesRequirementsGatherer),
+				typeof(StringRequirementsGatherer),
+				typeof(TypeRequirementsGatherer),
+			}
```

Comment:
If we're listing out the types anyway, we might as well just instantiate with `new()`.

Reply:
Oh! I didn't realize that's what this was doing, ha.